api/tests/phones_views_tests.py

```diff
@@ -326,11 +326,55 @@ def test_realphone_patch_invalid_verification_code(phone_user, mocked_twilio_cli
     mock_fetch.assert_not_called()
 
 
-def test_relaynumber_suggestions_bad_request_for_user_without_real_phone(phone_user):
-    real_phone = "+12223334444"
-    RealPhone.objects.create(user=phone_user, verified=True, number=real_phone)
+def test_relaynumber_post_with_existing_returns_error(phone_user, mocked_twilio_client):
+    number = "+12223334444"
     relay_number = "+19998887777"
-    RelayNumber.objects.create(user=phone_user, number=relay_number)
+    RealPhone.objects.create(user=phone_user, number=number, verified=True)
```

Comment:
This change looks weird. Basically, I saw that these 2 tests (`test_relaynumber_suggestions_bad_request_for_user_without_real_phone` and `test_relaynumber_post_with_existing_returns_error`) had the wrong names. So I switched the names so the test names would match what the test is actually doing. But this is how git interpreted that change. 🤷